### PR TITLE
Addition of new accounts in block list

### PR DESCRIPTION
**Pull request by dbertho**

### CODE REVIEW

File: blocklist.csv [hunk 1]
```diff
@@ -563,10 +630,11 @@
 125680703,SkodaFrance,ŠKODA France
 124176955,G5games,G5 Games
 123818293,DisneylandParis,Disneyland Paris
-123193555,EA_Benelux,EA Benelux
-123093566,datadoghq,|Datadog, Inc.|
+123193555,EA_Benelux,EA Benelux, a PTC Technology|
```
Comment (JusticeRage) [inline blocklist.csv:633]: This line seems a bit weird, looks like the merge displaced this bit from line 524

File: blocklist.csv [hunk 2]
```diff
@@ -576,24 +644,30 @@
...
 100004577,Orange_France,Orange France
+98414217,tousuncotefoot,OnaTousunCôtéFoot ⚽️
 98047213,Huawei,Huawei
 97226338,Credit_Agricole,Groupe Crédit Agricole
+95868974,MondeInformatiq,Le Monde Informatique
 95830798,EulerianTech,Eulerian
 93661938,ebayinccareers,Life at eBay
-92294003,Dove,Dove
+93608485,NissanMotor,Nissan Motor
+92294003,Dove,Dove, Inc.|
```
Comment (JusticeRage) [inline blocklist.csv:670]: Same here, I wonder if the `, Inc.` part doesn't come from above

File: blocklist.csv [hunk 3]
```diff
@@ -604,17 +678,19 @@
 84249568,AmazonKindle,Kindle
 83907610,GiltMAN,Gilt MAN
 82896397,INGFrance,ING France
-82347693,StarCraft_RU,StarCraft_RU
+82347693,StarCraft_RU,StarCraft_RU, Inc.|
```
Comment (JusticeRage) [inline blocklist.csv:681]: +1

File: blocklist.csv [hunk 4]
```diff
@@ -740,16 +826,17 @@
...
 19604690,Carbonite,Carbonite
+19398874,orangebusiness,Orange Business Services
 19292816,Betfair,Betfair
 19262500,zillow,Zillow
 19253334,pluralsight,Pluralsight
 19103481,Uber,Uber
 19017255,ericsson,Ericsson
 18956020,JackBox,Jack in the Box
-18807892,voyageprive,Voyage Privé FR
+18807892,voyageprive,Voyage Privé FR, a PTC Business|
```
Comment (JusticeRage) [inline blocklist.csv:839]: Same, seems to come from line 743

File: blocklist.csv [hunk 5]
```diff
@@ -765,23 +852,28 @@
...
 16573941,netflix,Netflix
 16557431,bouyguestelecom,Bouygues Telecom
 16334724,windowsdev,Windows Developer
-16043308,ChickfilA,|Chick-fil-A, Inc.|
+16144151,renaultgroup,Renault Group
+16096416,VogueFrance,Vogue France
+16043308,ChickfilA,|Chick-fil-A
 15996830,nielsen,Nielsen
-15991258,Xerox,Xerox
+15991258,Xerox,Xerox, Inc.|
```
Comment (JusticeRage) [inline blocklist.csv:876]: Probably another one

File: blocklist.csv [hunk 6]
```diff
@@ -224,7 +254,7 @@
 2420264664,DockersFrance,Dockers France
 2401980110,tiktok_us,TikTok
 2401682532,Spotify_PH,Spotify Philippines
-2399324190,AppsFlyerDev,AppsFlyer Engineering
+2399324190,AppsFlyerDev,AppsFlyer Engineering, LLC|
```
Comment (JusticeRage) [inline blocklist.csv:257]: The modified bit seems to belong to line 140

File: blocklist.csv [hunk 7]
```diff
@@ -162,21 +189,24 @@
...
+2874959663,Russia,Russia 🇷🇺
 2874803581,steamintech,Coffee + Tech
 2873526639,NOPLPoff,NOPLP Officiel
 2865691205,365ninja,365 Ninja
 2857454552,Uber_TX,Uber Texas
 2857406022,uber_phx,Uber Phoenix
 2848423391,ITI_Jobs,Infinite Technologie
 2834176185,WinamaxSport,Winamax Sport 🔞
-2832518219,NRM_inc,|NRM, Inc.|
+2832518219,NRM_inc,|NRM
```
Comment (JusticeRage) [inline blocklist.csv:200]: The ,Inc.| part seems to have been moved somewhere else, the CSV won't be valid without the matching "|"

**Review by JusticeRage — CHANGES_REQUESTED** [review]
Would you be able to confirm that the following accounts did indeed generate promoted content? I'm sure they generate annoying content, but I'd rather keep countries and newspapers out of the list unless they're really offenders:

@Russia
@CGTNFrancais
@GlblCtzn
@28minutes
@SciReports
@LePoint

If not, then I suggest removing them from this list. Beyond that, I've seen a few places where parts of the CSV seem to have been displaced and should probably be fixed!

**Comment by dbertho** [comment]
Closing pull request before fixing issues

**Comment by russia** [comment]
> Would you be able to confirm that the following accounts did indeed generate promoted content? I'm sure they generate annoying content, but I'd rather keep countries and newspapers out of the list unless they're really offenders:
> 
> @russia @CGTNFrancais @GlblCtzn @28minutes @SciReports @lepoint
> 
> If not, then I suggest removing them from this list. Beyond that, I've seen a few places where parts of the CSV seem to have been displaced and should probably be fixed!

?

**Pull request closed without merging**